docs/schema/guidance/repr-trusts.rst

```diff
@@ -0,0 +1,97 @@
+
+Example Scenarios 
+-----------------
+These examples show how statements could be connected to represent ownership networks involving trusts. Only the property:value pairs that are relevant to the scenario are shown in the statement blocks. 
+
+Scenario 1
+^^^^^^^^^^
+A trust with a beneficiary, trustee and settlor. The trust does not have a legal personality.
+
+.. figure:: ../../_assets/trust-scenario1.svg
+   :alt: entity statement labeled Trust A with entityType 'arrangement' and entitySubtype generalCategory 'trust.' Trust A is connected to person A by a relationship statement with interest type 'settlor', person B by a relationship statement with interest type 'trustee', and person C by a relationship statement with interest type 'beneficiary'
+   :figwidth: 100%
+   :align: center
+   
+Scenario 2
+^^^^^^^^^^
+A trust with a beneficiary, trustee and settlor and protector. The trust has a legal personality. 
+
+.. figure:: ../../_assets/trust-scenario2.svg
+   :alt: entity statement labeled Trust A with entityType 'legalEntity' and entitySubtype generalCategory 'trust.'  Trust A is connected to person A by a relationship statement with interest type 'settlor', person B by a relationship statement with interest type 'trustee', person C by a relationship statement with interest type 'beneficiary', and person D by a relationship statement with interest type 'protector'
+   :figwidth: 100%
+   :align: center
+   
+   
+Scenario 3
+^^^^^^^^^^
+A trust where the settlor and beneficiary are also both trustees. 
+
+.. figure:: ../../_assets/trust-scenario3.svg
+   :alt: Trust A is connected to person A by a relationship statement with interest types 'settlor' and 'trustee', person B by a relationship statement with interest type 'trustee', person C by a relationship statement with interest types 'beneficiary' and 'trustee'
+   :figwidth: 100%
+   :align: center
+
+Scenario 4
+^^^^^^^^^^
+A trust-like arrangement where local names are provided for the entityType and interestTypes. 
+
+.. figure:: ../../_assets/trust-scenario4.svg
+   :alt: entity statement labeled Trust A with entityType 'arrangement', entitySubtype generalCategory 'trust', and entitySubtype localTerm 'fiducie-sûreté.' Trust A is connected to person A by a relationship statement with interest type 'settlor' and interest details 'constituant', person B by a relationship statement with interest type 'trustee' and interest details 'fiduciaire', and person C by a relationship statement with interest type 'beneficiary' and interest details 'beneficiaire'
+   :figwidth: 100%
+   :align: center
+
+Scenario 5
+^^^^^^^^^^
+A trust that owns shares in a company. The beneficial owners of the trust are the beneficial owners of the company. Overarching relationship statements are represented by dotted lines. 
+
+See `representing beneficial ownership <repr-beneficial-ownership>`_ for guidance on representing indirect beneficial ownership.
```

Comment:
This link doesn't work. I think this will, though:

```suggestion
See :ref:`representing-bo` for guidance on representing indirect beneficial ownership.
```